proto/cosmos/bank/v1beta1/query.proto

```diff
@@ -63,6 +63,12 @@ service Query {
   rpc DenomOwners(QueryDenomOwnersRequest) returns (QueryDenomOwnersResponse) {
     option (google.api.http).get = "/cosmos/bank/v1beta1/denom_owners/{denom}";
   }
+
+  // SendEnabled queries for SendEnabled entries on a denomination.
+  // Denominations that don't have a specific SendEnabled entry, are not returned and would thus use the default (defined in Params).
```

Comment:
Can we please keep the proto + godocs wrapped at 80 chars 🙏

Reply:
> are not returned and would thus use the default (defined in Params).

What does this mean? The `Params.SendEnabled` field is deprecated and should not be used. What should be stated and implemented is that if a denom is not present, then the denom is allowed by default. Granted, we can also make the inverse assumption (i.e. NOT enabled by default).

Reply:
Ahh yes, that makes sense.